CHANGELOG.md

```diff
@@ -9,7 +9,10 @@ This file is used to list changes made in each version of the aws-parallelcluste
 **ENHANCEMENTS**
 
 **CHANGES**
-- WIP Perform default job-level scaling for all jobs, by reading job information from `SLURM_RESUME_FILE`.
+- Perform default job-level scaling for all jobs, by reading job information from `SLURM_RESUME_FILE`. Job-level scaling
```

Comment:
**NIT:** Perform job-level scaling by default for all jobs, using information in the `SLURM_RESUME_FILE`. ...